### PR TITLE
add Gluon v2020.1.3 releasenotes

### PR DESCRIPTION
**Pull request by blocktrron**
This adds the Releasenotes for the Gluon v2020.1.3 release.

### CODE REVIEW

**Comment by rotanid** [comment]
suggestion for the last bugfix:

`Fixes a regression which lead to 2.6MiB higher memory usage on the targets ar71xx-tiny and ramips-rt305x`

File: docs/releases/v2020.1.3.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which frequently change from single to multi-threaded operation.
+
+- Fixes a bug in the Linux kernel which leads to frequent reboots on MIPS platforms.
```
Comment (NeoRaider) [inline docs/releases/v2020.1.3.rst:9]: This was already fixed in v2020.1.2.

File: docs/releases/v2020.1.3.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which frequently change from single to multi-threaded operation.
```
Comment (NeoRaider) [inline docs/releases/v2020.1.3.rst:7]: "alternate between single- and multi-threaded operation" (the actual bug affected the change from multi- to single-threaded)

**Comment by NeoRaider** [comment]
I just noticed that `version` is set incorrectly in the `v2020.1.x` branch - it should be set to the latest release in the stable branch. Let's fix that in this PR as well.

File: docs/releases/v2020.1.3.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which alternate between single- and multi-threaded operation.
+
+- Fixes a regression which lead to around 2.5 MiB higher memory usage for ar71xx-tiny and ramips-rt305x targets.
```
Comment (NeoRaider) [inline docs/releases/v2020.1.3.rst:9]: Typo: "lead" -> "led"

**Comment by blocktrron** [comment]
@NeoRaider which `version` are you referring to?

**Comment by NeoRaider** [comment]
> @NeoRaider which `version` are you referring to?

Oops, I meant to write that this is about `docs/conf.py`.

**Comment by blocktrron** [comment]
@NeoRaider Hmm, to me the `v2020.1+` seems right for a v2020.1.3 release? Or am i looking at it the wrong way?

**Comment by NeoRaider** [comment]
We only set versions with a `+` in master usually, all past stable branches used a concrete version number there.

**Comment by blocktrron** [comment]
Okay, now i understand. Will change this.

File: docs/releases/v2020.1.3.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which alternate between single- and multi-threaded operation.
+
+- Fixes a regression which led to around 2.5 MiB higher memory usage for ar71xx-tiny and ramips-rt305x targets.
```
Comment (NeoRaider) [inline docs/releases/v2020.1.3.rst:9]: We should mention that this increases the firmware size by ~64K.

**Comment by rotanid** [comment]
now that the mr3020 fix has also been done, we can check for other cherry-picks again and get the release notes in shape for release, i guess.

some suggestions for backport:
- EA6500v3 device support (3 commits! but no patches)
- ed094bc68c7c702620864805dc462ff87eff8880

File: docs/releases/v2020.1.3.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which alternate between single-
```
Comment (rotanid) [inline docs/releases/v2020.1.3.rst:7]: maybe we want to link the github issues for bugfixes like we did in v2020.1.2 release notes?

**Comment by blocktrron** [comment]
@rotanid EA6350v3 has the primary-mac papertrail attached to i don't think we should backport.

The Mesh-firewall rule generation is pretty new in master, so I'm a bit hesitant to backport this as it's not broken feature-wise in v2020.1.x

**Comment by NeoRaider** [comment]
We can backport the firewall patch in the next point release, I agree that it is a bit too fresh.

**Comment by rotanid** [comment]
alright, then maybe link the issues like i suggested and we're good to go?

**Comment by blocktrron** [comment]
@rotanid done

I would like to do the release tomorrow if no one opposes this.

File: docs/releases/v2020.1.3.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+- Frequent reboots due to out-of-memory or high load due to memory pressure on weak hardware especially in larger
+  meshes (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
+
+  Optimizations in Gluon 2018.1 have significantly improved memory usage.
+  There are still known bugs leading to unreasonably high load that we hope to
+  solve in future releases.
```
Comment (rotanid) [inline docs/releases/v2020.1.3.rst:61]: we may want to remove this note as we have deprecated these devices already in v2020.1 ?

**Comment by rotanid** [comment]
> I would like to do the release tomorrow if no one opposes this.

can be done anytime from my POV

File: docs/releases/v2020.1.3.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+  This may lead to issues in environments where a fixed MAC address is expected (like VMware when promiscuous mode is
+  disallowed).
+
+- Inconsistent respondd API (`#522 <https://github.com/freifunk-gluon/gluon/issues/522>`_)
+
+  The current API is inconsistent and will be replaced eventually. The old API will still be supported for a while.
+
+- Frequent reboots due to out-of-memory or high load due to memory pressure on weak hardware especially in larger
+  meshes (`#1243 <https://github.com/freifunk-gluon/gluon/issues/1243>`_)
```
Comment (mweinelt) [inline docs/releases/v2020.1.3.rst:57]: The issue is closed, we should therefore drop this.

File: docs/releases/v2020.1.3.rst (hunk 6 above)
Comment (blocktrron) [reply docs/releases/v2020.1.3.rst:61]: Removed as suggested by @mweinelt

**Pull request merged**